approval_polls/templates/approval_polls/my_polls.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div  class='row-fluid top-buffer'>
+    <div class='col-md-12'>
+      <h3> My Polls </h3>
+    </div>
+  </div>
+  {% if latest_poll_list %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+        {% for poll in latest_poll_list %}
+          <div class='well well-sm'>
+            <a href="{% url 'approval_polls:detail' poll.id %}">{{ poll.question }}</a>
+            <a class='pull-right btn btn-danger btn-xs right-buffer'> remove </a>
```

Comment:
Could you remove the right-buffer class from this link as it looks ok without it too and less code is easier to maintain :smile: